portfolio/src/main/webapp/script.js

```diff
@@ -36,10 +36,10 @@ function showSlides(n) {
     }
 }
 
-function onLoad(){
-    movieRecommender();
-    getComments();
-}
+document.getElementById('movie-container').addEventListener("load", movieRecommender());
+window.addEventListener("load", getComments());
+document.getElementById('map').addEventListener("load", createMap());
```

Comment:
I don't think you have to add brackets for these event listener registrations. You want to register with the function object, not the result of the function invocation. By adding the brackets, you're calling the function and executing it immediately even before the element is loaded.

Reply:
Just double checking, for the brackets, are you referring to the brackets after function calls? For example, I should have addEventListener("load", getComments) instead of addEventListener("load", getComments() )?

Reply:
Yes that's what I meant.